mailbox/api/src/main/java/org/apache/james/mailbox/model/QuotaThreshold.java

```diff
@@ -17,13 +17,11 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.mailbox.quota.model;
+package org.apache.james.mailbox.model;
```

Comment:
I think that you favor reuse too much vs context bounded domain concepts.
Crossing a quota threshold in the case of quota mailing is not the same concept as defining the range of a request in a REST API.
In this case, we should not try to reuse the object (and BTW, It probably makes more sense to name the boundaries of the request QuotaRange or QuotaBoundary because it intends to limit the query scope)